pkg/ddevapp/global_dotddev_assets/commands/host/sequelace

```diff
@@ -10,7 +10,7 @@
 
 DATABASE="${1:-db}"
 
-query="mysql://root:root@127.0.0.1:${DDEV_HOST_DB_PORT}/${DATABASE}"
+query="mysql://root:root@${DDEV_HOSTNAME}:${DDEV_HOST_DB_PORT}/${DATABASE}"
```

Comment:
I would like to suggest `${DDEV_PROJECT}`. Unique and simple because it is the name of your project.

```suggestion
query="mysql://root:root@${DDEV_PROJECT}:${DDEV_HOST_DB_PORT}/${DATABASE}"
```

Reply:
DDEV_PROJECT won't work AFAICT because it's just the project name. This needs to be the domain.

Reply:
But the bottom line is you want something descriptive that is guaranteed to resolve to 127.0.0.1. It would be better if SequelAce had a command-line arg. Are you sure it doesn't? They're very responsive...

Reply:
It sounds like it doesn't: https://github.com/Sequel-Ace/Sequel-Ace/issues/189
And anyway, this isn't using SequelAce directly, but using the `open` command, passing SequelAce to its -a flag. It doesn't look like we can pass more to the app: https://ss64.com/osx/open.html

Reply:
Probably `${DDEV_PROJECT}.${DDEV_TLD}` will always work for you.

Reply:
DDEV_PRIMARY_URL won't work because it includes the http://

Reply:
DDEV_PROJECT -- can we guarantee that there is always a domain of the form  ${DDEV_PROJECT}.${DDEV_TLD} for a project? Could the user have configured it differently?

Reply:
I believe it is a reliable way.

Or use `${DDEV_SITENAME}.${DDEV_TLD}`, the same but perhaps more descriptive.

Also, you can check the values of these variables yourself with a custom command:

- https://github.com/ddev/ddev/issues/3636#issuecomment-1055509594

Reply:
The docs for DDEV_SITENAME are identical to those for DDEV_PROJECT:

> Project name, like d8composer

Reply:
I think DDEV_SITENAME and DDEV_PROJECT will always be the same. 

> can we guarantee that there is always a domain of the form ${DDEV_PROJECT}.${DDEV_TLD} for a project? 

DDEV insists on either having it in hosts or via DNS.

You could also easily use `${DDEV_PRIMARY_URL#*//}` which would be fine. 

The only problem is the situation where there's no internet and somebody hasn't already been forced to `ddev start` so the hostname hasn't been added to /etc/hosts. But in that case their browser doesn't work either.

Reply:
Fixed.